x-pack/plugins/security_solution/server/lists_integration/endpoint/validators/host_isolation_exceptions_validator.ts

```diff
@@ -60,18 +60,20 @@ export class HostIsolationExceptionsValidator extends BaseValidator {
     return item.listId === ENDPOINT_HOST_ISOLATION_EXCEPTIONS_LIST_ID;
   }
 
-  protected async validateHasWritePrivilege(): Promise<void> {
-    return super.validateHasPrivilege('canWriteHostIsolationExceptions');
-  }
+  // TODO: 8.7 rbac
+  // protected async validateHasWritePrivilege(): Promise<void> {
+  //   return super.validateHasPrivilege('canWriteHostIsolationExceptions');
+  // }
 
-  protected async validateHasReadPrivilege(): Promise<void> {
-    return super.validateHasPrivilege('canReadHostIsolationExceptions');
-  }
+  // TODO: 8.7 rbac
+  // protected async validateHasReadPrivilege(): Promise<void> {
+  //   return super.validateHasPrivilege('canReadHostIsolationExceptions');
+  // }
 
   async validatePreCreateItem(
     item: CreateExceptionListItemOptions
   ): Promise<CreateExceptionListItemOptions> {
-    await this.validateHasWritePrivilege();
+    // TODO add this to 8.7 rbac await this.validateHasWritePrivilege();
```

Comment:
Should we add the `validateCanManageEndpointArtifacts` here as it was before?

Reply:
Yeah I missed that. Will add it

Reply:
Hold on! I don't see that `validateCanManageEndpointArtifacts` was ever included in the `preCreateItem` validation. We can add it now if that makes sense @dasansol92

Reply:
You are right, it was used in get methods since `preUpdateItem` and `preCreateItem` have the `canIsolateHost` check. You can check it here: https://github.com/elastic/kibana/pull/144021/files